.github/workflows/tests.yml

```diff
@@ -144,6 +144,11 @@ jobs:
         run: |
           flake8 . --config=setup.cfg --count --statistics
 
+      - name: mypy
+        id: black
+        run: |
+          mypy histolab .
```

Comment:
we should move all these steps outside the matrix, they should run only once 😊

![https://media.giphy.com/media/MC6eSuC3yypCU/giphy.gif](https://media.giphy.com/media/MC6eSuC3yypCU/giphy.gif)

Reply:
good point